server/data/localMockData/learnerGoalsMock.ts

```diff
@@ -1,20 +1,19 @@
 import { LearnerGoals } from '../../interfaces/learnerGoals'
 
-export const LearnerGoalsMock: LearnerGoals = {
-  prn: 'G6123VU',
-  employmentGoals: ['string'],
-  personalGoals: ['string'],
-  longTermGoals: ['string'],
-  shortTermGoals: ['string'],
+const aValidLearnerGoals = (options?: {
```

Comment:
`LearnerGoalsMock` and `LearnerGoalsMockB` have been refactored into `aValidLerarnerGoals` function that takes an options object to control how it is constructed